Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "kakarot-rpc-adapter"
-version = "0.1.0"
+version = "0.0.1"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = "1.0.65"
-futures = "0.3.25"
-tarpc = { version = "0.30.0", features = ["tokio1"] }
-tokio = { version = "1.21.2", features = ["rt", "rt-multi-thread", "macros"] }
+clap = { version = "4.0.32", features = ["derive"] }
+rust-crypto = "0.2.36"
+jsonrpc-http-server = "18.0"
```

Comment:
why this framework:)? I understand there are more standard choices for building a json rpc server

Reply:
`json-http-server` because it is the same lib used by the Parity client, `rust-crypto` for hex, bytes, hashing, etc... and `clap` is required with `json-http-server`